src/MyEventEmitter.ts

```diff
@@ -0,0 +1,150 @@
+'use strict';
+
+class Listener {
+  eventName;
+  callback;
+  once;
```

Comment:
any; 🥲

Reply:
actually, that was `.js` file. I wanted to add TS but after getting a few conflict errors (with already installed packages) I got rid of this idea not to waste time.

Or do you mean to add the default value here? i.e. `once = false`

Reply:
No, I meant types. At the moment of writing it was TS.